.claude/skills/create-gatherer/SKILL.md

```
@@ -0,0 +1,423 @@
+---
+name: create-gatherer
+description: Create a new gathering function including implementation, tests, sample archive data, registration, and docs regeneration
+---
+
+# create-gatherer
+
+Create a new gathering function in the insights-operator, following the project's established patterns exactly.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)